### DISCUSSION TITLE
4.2 [Affiliated packages] confusing parenthesis usage in citations

### DISCUSSION THREAD

**Discussion by dhomeier** [astropy/astropy-v5.0-paper#123] [Text comments] [answered]
In 4.2, especially the first listing of packages following `expansion of affiliated packages`, a lot of citations are using, incorrectly I believe, `\cite{}` rather than `\citep{}`, which makes for awkward reading (I have not looked extensively elsewhere; sorry for having managed to miss the mail for final checks first and then being distracted by feature freeze).

**Comment by adrn** [comment, marked as answer]
Fixed in a commit I will push soon
